### PR TITLE
reflect 2.3.7 release in changelog

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: blackbox/docs/appendices/release-notes/2.3.7.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Changelog
+=========
+
+Changes
+-------
+
+- Table functions no longer require any permissions to be used. (Enterprise
+  only)
```
Comment (nslater) [inline blackbox/docs/appendices/release-notes/2.3.7.rst]: "only.)"

File: blackbox/docs/appendices/release-notes/2.3.7.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Fixes
+-----
+
+- Fixed an issue in binary encoding/decoding of array types in Postgres wire
```
Comment (nslater) [inline blackbox/docs/appendices/release-notes/2.3.7.rst]: "encoding and decoding"

Comment (nslater) [reply blackbox/docs/appendices/release-notes/2.3.7.rst]: "PostgreSQL wire"

File: blackbox/docs/appendices/release-notes/2.3.7.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  protocol affecting the ``npgsql`` driver.
+
+- As Windows does not support sending signals, stop attempting to handle the
```
Comment (nslater) [inline blackbox/docs/appendices/release-notes/2.3.7.rst]: "Microsoft Windows"

File: blackbox/docs/appendices/release-notes/2.3.7.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  ``USR2`` signal on Windows platforms as it just pollutes the log with an
```
Comment (nslater) [inline blackbox/docs/appendices/release-notes/2.3.7.rst]: drop "just"

File: blackbox/docs/appendices/release-notes/2.3.7.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  unnecessary stacktrace.
+
+- Fixed a performance regression which occurred in queries where a literal was
+  compared against an array column of a different type. For example: ``SELECT *
+  FROM t1 WHERE 1 = ANY(int_arr_column)`` <- array column would be converted to
```
Comment (nslater) [inline blackbox/docs/appendices/release-notes/2.3.7.rst]: put example on new line, indented

reword and convert "<- array column would" into new para

File: blackbox/docs/appendices/release-notes/2.3.7.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  long although 1 could have been converted to integer.
+
+- Fixed an issue where the order of the relations in ``FROM`` would expose
```
Comment (nslater) [inline blackbox/docs/appendices/release-notes/2.3.7.rst]: "in a ``FROM`` clause"

File: blackbox/docs/appendices/release-notes/2.3.7.rst [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  fields between the relations. An error was thrown during execution of the
+  statements.
+
+- Fixed incorrect HTTP responses on the BLOB API of subsequent requests after
+  an error response occurred, e.g. 404 Not Found.
+
+- Fixed an issue that caused ``ANY`` to not work correctly on nested arrays.
+
+- Fixed an issue that would cause a NullPointerException to be thrown when
```
Comment (nslater) [inline blackbox/docs/appendices/release-notes/2.3.7.rst]: ``NullPointerException``

**Comment by chaudum** [comment]
retest this please

**Pull request merged**